contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -70,6 +70,50 @@ reasonable production worthy environments:
 
+### Windows & Linux Considerations
+
+As of v1.14, Windows is a stable feature, but not yet included in conformance
+testing. In preparation for this, a large number of conformance tests are
+already included in Windows testing. You can see what tests have already pass by
+looking at TestGrid for results of Windows tests running on 
+[Azure](https://testgrid.k8s.io/sig-windows#aks-engine-azure-windows-master) and 
+[GCE](https://testgrid.k8s.io/sig-windows#gce-windows-master)).
+
+Generally speaking, the goals are to:
+
+- Make sure tests that are already passing remain passing. If new OS-specific
+functionality is added, it should be in a new test.
+- Ensure that new tests covering Linux-specific functionality are tagged with `[LinuxOnly]` 
+(see: [Kinds of Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests),
+  - Give future reviewers a reference to an active issue or documentation clarifying why a test
+cannot run on Windows.
+
+The tests that are running today:
+
+- Rely only on container images that already have a multi-architecture manifest
+including Windows versions, or have been ported by SIG-Windows
+(see [kubernetes-sigs/windows-testing/images](https://github.com/kubernetes-sigs/windows-testing/tree/master/images)
+- Do not depend on any functionality is different or not available on Windows. The full list
+is available in the Windows Kubernetes docs under [api](https://kubernetes.io/docs/setup/windows/intro-windows-in-kubernetes/#api). 
+A brief summary is included here as a starting point. If the docs are insufficient
+or there are more questions, please contact #SIG-Windows on Slack to get another
+reviewer.
+
+Some of the most common differences to watch for are:
+- Most user & security context options are not supported since Windows does not
+use POSIX-style users, permissions or ACLs
+- Privileged containers and host networking modes are not supported. Containers are always isolated.
+- Only NTFS is supported. Volume mounts specifying other filesystems (ext4, xfs) or mediums (memory) are not supported
+  - Projected mounts and permission masks are not supported
+  - Mappings of individual files are not supported. This means some features that rely on it such as 
+V1.Container.terminationMessagePath or writing service entries to `/etc/hosts` also do not work.
+- Network and DNS settings must be passed through CNI. Windows does not use `/etc/resolv.conf` 
+and stores networking settings in the Windows registry instead
+- Windows treats all DNS lookups with a `.` to be FQDN, not PQDN.
+- Windows uses job objects or Hyper-V for pod isolation and resource controls, not CGroups. These are managed implicitly by Docker or ContainerD, not by the kubelet. Do not check properties of CGroups as pass/fail criteria.
+- ICMP only works between pods on the same network, and are not routable to external networks. TCP/UDP are routable.
+
```

Comment:
So, the information is mostly correct, I've made a few comments above about that.

But, my concern is that some of the differences / criteria listed are quite vague. In the PR I've made [1], I've listed concrete cases and examples I've encountered while Windows-ifying tests, order by number of occurences (most common cases first, it's easier to remember). I have also listed some items that are not on this list, so I think they should copied over. 

[1] https://github.com/kubernetes/community/pull/3522/files

Reply:
I merged the lists and grouped them into container options / storage / networking